### PR TITLE
Core weekly 39

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add CW 39 of 2020

### CODE REVIEW

File: news/_posts/core-weekly/2020-09-29-coreweekly-39-2020.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+* [#739](https://github.com/PrestaShop/docs/pull/739): Correcting functions name associated to http verb. Thank you [@moncef-essid](https://github.com/moncef-essid)
+* [#738](https://github.com/PrestaShop/docs/pull/738): Fix path error in template layouts page. Thank you [@moncef-essid](https://github.com/moncef-essid)
+* [#737](https://github.com/PrestaShop/docs/pull/737): Mention jQuery.attr has changed in 1.7.7, by [@matks](https://github.com/matks)
+
+
+### Core Weekly Generator tool
+* [#51](https://github.com/PrestaShop/core-weekly-generator/pull/51): Dont use zero-padded decimal for day number when using week number input, by [@matks](https://github.com/matks)
+
+
+### ps_legalcompliance
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-09-29-coreweekly-39-2020.md]:
```suggestion
### Legal Compliance
```

Comment (matks) [reply news/_posts/core-weekly/2020-09-29-coreweekly-39-2020.md]: Indeed I missed this one

**Review by Progi1984 — APPROVED** [review]

**Comment by ttoine** [comment]
did you add PS Legalcompliance to the list of the generator?

**Review by ttoine — APPROVED** [review]

**Comment by ttoine** [comment]
thanks @matks

**Pull request merged**